src/module/LayerZeroModule.sol

```diff
@@ -185,6 +193,9 @@ contract LayerZeroModule is Admin, Initializable, CrossChainMessageInterface, La
     require(gasLimit < gasParameters.maxGasLimit, "HOLOGRAPH: gas limit over max");
     (uint256 nativeFee, ) = lz.estimateFees(lzDestChain, address(this), crossChainPayload, false, adapterParams);
     hlgFee = ((gasPrice * gasLimit) * dstPriceRatio) / (10 ** 10);
+    if (toChain == uint32(7) || toChain == uint32(4000000015)) {
```

Comment:
Maybe adding a comment on these chain ids would be helpful so you don't need to look them up if you haven't memorized all the chain ids 😅

Reply:
especially for the holographId

Reply:
Good point, made a commit to address this.

Reply:
👍